apps/condo/domains/miniapp/schema/SendAppPushMessageService.js

```diff
@@ -71,7 +107,7 @@ const SendAppPushMessageService = new GQLCustomSchema('SendAppPushMessageService
         },
         {
             access: true,
-            type: 'input SendAppPushMessageData { body: String!, title: String, B2CAppContext: String }',
+            type: 'input SendAppPushMessageData { body: String!, title: String, B2CAppContext: String, resident: ResidentWhereUniqueInput, property: PropertyWhereUniqueInput, unitType: String, unitName: String }',
```

Comment:
It's strange you should add some `meta` field for any app-specific context and don't add resident, property or any other fields

Reply:
fixed